crates/cargo-platform/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "cargo-platform"
 version = "0.1.5"
 edition.workspace = true
 license.workspace = true
+rust-version = "1.70.0"  # MSRV:3
```

Comment:
This comment seems like a magic and is prone to remove? Do we have a way to avoid a remove even when maintainers aren't familiar with the msrv CI job?

Reply:
The only thing I could think of is to add additional text after saying this is a renovatebot marker.

Thoughts?